app/transforms/registration-responses.ts

```diff
@@ -17,7 +18,7 @@ export default class RegistrationResponsesTransform extends Transform {
         return mapKeysAndValues(
             obj,
             key => deserializeResponseKey(key),
-            value => value,
+            value => fixSpecialChars(value as string),
```

Comment:
Revert changes to `fix-special-chars` and:
```suggestion
            value =>  (typeof value === 'string' ? fixSpecialChars(value) : value),
```